packages/common/src/pipes/number_pipe.ts

```diff
@@ -268,7 +268,7 @@ export class CurrencyPipe implements PipeTransform {
       display = display ? 'symbol' : 'code';
     }
 
-    let currency: string = currencyCode || this._defaultCurrencyCode;
+    let currency: string = currencyCode;
```

Comment:
Please revert this change:

```suggestion
    let currency: string = currencyCode || this._defaultCurrencyCode;
```